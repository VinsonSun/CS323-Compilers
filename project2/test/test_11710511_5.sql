int test5() {
  int a = 1;
  char c = a;
  return c;
}
