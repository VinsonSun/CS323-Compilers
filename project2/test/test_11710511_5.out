Error type 5 at Line 3: unmatching type on both sides of assignment
Error type 8 at Line 4: incompatiable return type
